DataFormats/FTLRecHit/interface/FTLRecHit.h

```diff
@@ -41,6 +41,24 @@ class FTLRecHit {
   FTLRecHit(
       const DetId& id, uint8_t row, uint8_t column, float energy, float time, float timeError, uint32_t flagBits = 0);
 
+  FTLRecHit(const DetId& id,
+            float energy,
+            float time,
+            float timeError,
+            std::pair<float, float> position,
+            float positionError,
+            uint32_t flagBits = 0);
+
+  FTLRecHit(const DetId& id,
+            uint8_t row,
+            uint8_t column,
+            float energy,
+            float time,
+            float timeError,
+            std::pair<float, float> position,
+            float positionError,
+            uint32_t flagBits = 0);
```

Comment:
Hi,
I kept those constructors because I am not sure whether they are used by some other code.
If no code is using them I can just remove them.
Thanks.

Reply:
@Ang-Li-95 this is not the right approach in general when adding new features to CMSSW. Our goal should always be to have consistent physical output. If these constructors are being used by other code, you, as the developer, need to know that. Your change might make it so that using these constructors no longer provides valid output, in which case that other code will need to be updated.

Reply:
Hi,
Cool I will remove the old version of constructors and some other functions that are updated.
Thanks!